web/src/containers/ApdSummary.test.js

```diff
@@ -1,111 +1,166 @@
-import { shallow } from 'enzyme';
 import React from 'react';
+import { renderWithConnection, screen } from '../shared/apd-testing-library';
```

Comment:
Nice switch.

Meta note: It's interesting how libraries have different philosophies on testing. For example in the "add a year test," the enzyme test focuses on a method being called. Whereas in the react-testing-library one we test the UI only. Which is probably _more_ appropriate for react.